website/public/js/controllers/groupsCtrl.js

```diff
@@ -12,7 +12,7 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
     $scope.isMemberOfRunningQuest = function(userid, group) {
       if (!group.quest || !group.quest.members) return false;
       if (!group.quest.active) return false; // quest is pending, not started
-      return group.quest.members[userid];
+      return userid in group.quest.members && group.quest.members[userid] != false;
```

Comment:
When adding tests, i found that the behavior of this method and isMemberOfPendingQuest was inconsistent.  This should be functionally equivalent to what was there before but also handles undefined.  It is a safer check overall right?

In reality these two methods should probably be refactored for code reuse, since they are almost identical other than the active check.